src/odb/include/odb/db.h

```diff
@@ -4125,11 +4133,23 @@ class dbTrackGrid : public dbObject
   /// Get the "ith" "X" grid pattern.
   ///
   void getGridPatternX(int i, int& origin_x, int& line_count, int& step);
+  void getGridPatternX(int i,
```

Comment:
Are you adding an overload to avoid updating client code?

swig/common/dbtypes_common.i has an OUT pattern for this method.  You'll need to add another for the new APIs.

Reply:
to avoid changing a bunch of code, but also since the tools don't support the mask information, it seems like it would be easier to just access the information needed by the tool.